src/FastAI.jl

```diff
@@ -29,7 +28,11 @@ export valid
 
 export DummyCallback
 export ProgressCallback
-export Recorder
+
+export TrainLossRecorder
+export ValidateLossRecorder
+export TrainSmoothLossRecorder
+export ValidateSmoothLossRecorder
```

Comment:
I think @ToucheSir's comment about type names means that here we should export the module `Recorder` but not the types within it. Then in user code, a person needs to write `Recorder.TrainLoss` to get the training loss recorder. I agree that this will look cleaner than having "Recorder" at the end of each type name.